netflix-roulette/src/pages/home/components/Catalog/components/GenresTabs/GenresTabs.styles.ts

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+import { Theme } from "../../../../../../utils";
+
+export const Tabs = styled.div`
+  display: flex;
+  height: 50px;
+`;
+
+export const Tab = styled.button`
```

Comment:
You seem to be using template string values interpolation (``.. ${ .. }.. ``) quite a lot in this particular styled component, so you may go for [object syntax](https://styled-components.com/docs/advanced#style-objects) to somewhat improve readability, e.g.:
```ts
export const Tab = styled.button(
   ({theme, isSelected}) => {
      const {
         fontSize,
         palette: {
            red,
            grey,
            white
         } = theme
      const borderBottom = isSelected
         ?   `3px solid ${red}`
         :    `1px solid ${grey}`      

      return {
         ..
         padding: `0 ${fontSize}`,
         ..
         borderBottom,
         ..
         color: white,
         ..
         '&:hover': {
            borderBottom: `3px solid ${red}`
         }
      }
   }
)
```